src/constants/routes.ts

```diff
@@ -227,11 +227,9 @@ export const getProjectSettingsRoute = (
   projectId: string,
   tab?: ProjectSettingsTabRoutes
 ) => {
-  if (!tab) {
-    return `${paths.project}/${projectId}/${PageNames.Settings}`;
-  }
-
-  return `${paths.project}/${projectId}/${PageNames.Settings}/${tab}`;
+  const encodedProjectId = encodeURIComponent(projectId);
```

Comment:
The description says we don't need to encode the URL, should this be included

Reply:
Nice -- since the ticket mentions that we'll work with project admins to change their project identifiers, can we update the comment to note that we'll delete this when that is done?